ios/RNFirebase.podspec

```diff
@@ -14,9 +14,12 @@ Pod::Spec.new do |s|
   s.source              = { :git => "https://github.com/invertase/react-native-firebase.git", :tag => "v#{s.version}" }
   s.social_media_url    = 'http://twitter.com/invertaseio'
   s.platform            = :ios, "9.0"
-  s.source_files        = 'RNFirebase/**/*.{h,m}'
-  s.dependency          'React'
-  s.dependency          'Firebase/Core'
+  s.default_subspecs    = 'Core'
+  s.subspec 'Core' do |cs|
+    cs.dependency 'React'
+    cs.dependency 'Firebase/Core'
+    cs.source_files = 'RNFirebase/**/*.{h,m}'
+  end
```

Comment:
Glad I found this PR requesting similar I was about to do it!

Actually the pod setup has a tiny but damaging bug. There's a subspec created for crashlytics however the main spec is adding this files `RNFirebaseCrashlytics.h` and `RNFirebaseCrashlytics.m` by default causing 💥 💥 💥  because those files will want to refer to the Crashlytics framework resulting in undefined symbols. Like;

```
Undefined symbol: _OBJC_CLASS_$_CLSStackFrame
Undefined symbol: _OBJC_CLASS_$_Crashlytics
Undefined symbol: _CLSNSLog
```

The correct approach is by excluding those fabric/crashlytics files and adding them only in the subspec. Resulting in this setup;

```
s.default_subspec     = 'Core'

s.subspec 'Core' do |cs|
    cs.source_files        = 'RNFirebase/**/*.{h,m}'
    cs.exclude_files       = 'RNFirebase/fabric/**/*.{h,m}'

    cs.dependency          'React'
    cs.dependency          'Firebase/Core'
end

s.subspec 'Crashlytics' do |cs|
    cs.source_files = 'RNFirebase/fabric/**/*.{h,m}'

    cs.dependency 'Fabric'
    cs.dependency 'Crashlytics'
end
```

Hopefully we can get this PR soon 😄

Reply:
good catch, thanks!
pushed update